src/typings.d.ts

```diff
@@ -1,2 +1,5 @@
 declare module '@department-of-veterans-affairs/component-library/Date';
-declare module '@department-of-veterans-affairs/component-library/dist/react-bindings';
\ No newline at end of file
+declare module '@department-of-veterans-affairs/component-library/CheckboxGroup';
```

Comment:
If we can get `VaCheckboxGroup` from the component library react bindings we may not need this here.